### DISCUSSION TITLE
I hope to display a draggable button before each paragraph in the editor, and when dragging the button, I can select the paragraph for dragging.

### DISCUSSION THREAD

**Discussion by kailiming** [ueberdosis/tiptap#5355] [General Questions] [answered]
![image](https://github.com/user-attachments/assets/a2a38efd-a302-4dec-b08c-814cd95b4c46)

### I hope to display a draggable button before each paragraph in the editor, and when dragging the button, I can select the paragraph for dragging.
Could you please give me some advice?

**Comment by nperez0111** [comment, marked as answer]
If you use tiptap pro this is the drag handle extension. You can see its usage in the notion like template. Unfortunately we have not gotten around to writing the docs specifically for it yet
